backend/src/routes/user.ts

```diff
@@ -0,0 +1,74 @@
+/**
+ * This class contains routes that will create and get
+ * new users
+ */
+import express, { NextFunction, Request, Response } from "express";
+import { Mentee } from "../models/mentee";
+import { Mentor } from "../models/mentor";
+import { validateMentee, validateMentor } from "../middleware/validation";
+import { createUser } from "../services/auth";
+
+const router = express.Router();
+
+/**
+ * This is a post route to create a new mentee. It will first validate
+ * the mentee to ensure they follow standard listed below before adding them to
+ * mongoDB and Firebase
+ *
+ * Mentee: {type: string, name: string, email: string,password: string}
+ */
+router.post(
```

Comment:
<img width="200" alt="image" src="https://user-images.githubusercontent.com/56745367/219135274-2d0bf867-a35f-4fc7-9a23-d6530a877e58.png">

There are several fields still pending from the figma. This route will be called when the submit button is pressed and it will send all the data collected in the sign-up flow to here. Can you add those fields to the route and model?

Reply:
add grade, topics of interest, career interests, and what do you hope question

Reply:
https://github.com/TritonSE/ALUM-Mobile-Application/issues/46